docs/source/devenv-install.md

```diff
@@ -7,23 +7,28 @@ the RPM packages provided by the [Copr](https://copr.fedorainfracloud.org/coprs/
 build system, which automatically builds packages with every commit merged into master.
 Packages are built for EPEL and Fedora.
 
-```shell
-    yum install -y yum-utils
-    yum-config-manager --add-repo https://copr.fedorainfracloud.org/coprs/g/leapp/leapp-devel/repo/epel-7/group_leapp-leapp-devel-epel-7.repo
-```
+* On CentOS/RHEL:
+  ```shell
+  $ yum install -y yum-utils
+  $ yum-config-manager --add-repo https://copr.fedorainfracloud.org/coprs/g/leapp/leapp-devel/repo/epel-7/group_leapp-leapp-devel-epel-7.repo
+  ```
+* On Fedora (replace `30` in the URL below if you're on another version of Fedora, but make sure such a repository exists on the Copr):
+  ```shell
+  $ dnf config-manager --add-repo https://copr.fedorainfracloud.org/coprs/g/leapp/leapp-devel/repo/fedora-30/group_leapp-leapp-devel-fedora-30.repo
```

Comment:
I think that this is not the best approach. I prefer way described directly in the copr:
```shell
 # dnf copr enable @leapp/leapp-devel
```

With that command, you don't need to point to the specific version of fedora.